### PR TITLE
chore(server): set superdesk/eve version via  commit

### PR DESCRIPTION
**Pull request by petrjasek**

### CODE REVIEW

**Comment by actionless** [comment]
LGTM

File: server/requirements.txt [hunk 1]
```diff
@@ -40,6 +40,6 @@ python3-ldap==0.9.5.3
 behave==1.2.5
 wooper==0.4.2
 
-git+git://github.com/superdesk/eve@sync-upstream#egg=Eve==0.6-dev
+git+git://github.com/superdesk/eve@e49cca#egg=Eve==0.6-dev
```
Comment (jey07ro) [inline server/requirements.txt]: why is enought just e49cca when the full hash is e49cca9e48b4d33fe362305d8463314113642a6f?

File: server/requirements.txt [hunk 2]
```diff
@@ -40,6 +40,6 @@ python3-ldap==0.9.5.3
 behave==1.2.5
 wooper==0.4.2
 
-git+git://github.com/superdesk/eve@sync-upstream#egg=Eve==0.6-dev
```
Comment (nistormihai) [inline server/requirements.txt]: how is enough just e49cca when the full hash is e49cca9e48b4d33fe362305d8463314113642a6f?

Comment (plamut) [inline server/requirements.txt]: GIT is smart enough to find a specific commit based only on the first few characters of the hash (provided that the combination is unique, of course).

Comment (ioanpocol) [inline server/requirements.txt]: it's a kind of magic :)

Comment (nistormihai) [inline server/requirements.txt]: smart is always better, we need to make smart things also :)
Thanks for help Peter.

Comment (plamut) [inline server/requirements.txt]: YW :)

**Pull request merged**